src/components/organisms/NewProfileModal/ProfileModal/index.ts

```diff
@@ -0,0 +1 @@
+export { ProfileModal } from "components/organisms/NewProfileModal/ProfileModal/ProfileModal";
```

Comment:
```suggestion
export { ProfileModal } from "./ProfileModal";
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1934/commits/966a98a7c5783a85a2cf8361a69adb3bc783f26b